R/data-repository.R

```diff
@@ -0,0 +1,56 @@
+#' @title DataRepository
+#' @docType class
+#' @description  An object typically holding observed data

```

Comment:
Description is identical with that of `DataColumn`- please change it to something more distinguishable.